src/modules/tests/test-data/cat-a-mod1/avoidance/avoidance.selector.ts

```diff
@@ -3,6 +3,6 @@ import { TestData, Avoidance } from '@dvsa/mes-test-schema/categories/AM1';
 export const getAvoidance = (testData: TestData): Avoidance => testData.avoidance;
 
 export const getAvoidanceAttempted = (avoidance: Avoidance) : boolean => {
-  if (avoidance.firstAttempt || avoidance.secondAttempt) return true;
+  if (avoidance.firstAttempt !== undefined || avoidance.secondAttempt !== undefined) return true;
```

Comment:
you can return the expression evaluation and remove the if statement

Reply:
Yes correct! We can now before there was a typical issue with them typed as strings but compared to undefiend we can use a lamba statment